examples/C++/using_installed_STIR/demo_create_image.cxx

```diff
@@ -0,0 +1,60 @@
+#include "stir/Shape/EllipsoidalCylinder.h"
+#include "stir/PatientPosition.h"
+#include "stir/ImagingModality.h"
+#include "stir/CartesianCoordinate3D.h"
+#include "stir/IndexRange3D.h"
+#include "stir/Succeeded.h"
+#include "stir/IO/write_to_file.h"
+#include "stir/VoxelsOnCartesianGrid.h"
+#include <iostream>
+
+int main()
+{
+
+  std::string output_filename("test.hv");
+  int output_image_size_x = 100;
+  int output_image_size_y = 104;
+  int output_image_size_z = 20;
+  float output_voxel_size_x = 5.F;
+  float output_voxel_size_y = 4.F;
+  float output_voxel_size_z = 3.F;
+  double image_duration = 100.; // secs
+  double rel_start_time = 1; //secs
+
+  auto exam_info_sptr = std::make_shared<stir::ExamInfo>(stir::ImagingModality::PT);
```

Comment:
I don't love the use of `auto` but its fine

Reply:
I start to like it more... Less verbose than
```c++
std::shared<stir::ExamInfo> exam_info_sptr(new ExamInfo(stir::ImagingModality::PT));
```